app/models/form_handler.rb

```diff
@@ -79,12 +79,12 @@ def in_crossover_period?(now: Time.zone.now)
 
   def lettings_in_crossover_period?(now: Time.zone.now)
     forms = lettings_forms.values
-    forms.count { |form| form.start_date < now && now < form.end_date } > 1
+    forms.count { |form| form.start_date <= now && now < form.end_date } > 1
```

Comment:
think this was a bug that got missed previously, can't think of a reason this shouldn't be <=

Reply:
`.between?`

Reply:
between is inclusive at both ends of a range I believe, think I could use ... though

Reply:
Ah no ignore me, between is fine here I think, thanks!